portieris.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: portieris
+  version: 0.13.19
+  epoch: 0
+  description: A Kubernetes Admission Controller for verifying image trust.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/IBM/portieris.git
+      tag: v${{package.version}}
+      expected-commit: d0705039d060ab11b8fa598b79647ac0008af57e
+
+  - runs: |
+      VERSION=v${{package.version}} make portieris
+
```

Comment:
did you try using go/build ?

Reply:
I did not.  Is there a documented standard for when we should be using go/build vs make?  My only other experience with Wolfi packages is another Go package that uses make for its build.

Reply:
Works with go/build

Reply:
There is not a documented standard.